conan/tools/env/environment.py

```diff
@@ -568,6 +580,9 @@ def save_script(self, filename):
         else:
             self.save_sh(path)
 
+        if self._conanfile.conf.get("tools.env:dotenv", check_type=bool):
+            self.save_dotenv(f"{name}.env")
```

Comment:
```suggestion
        self.save_dotenv(f"{name}.env")
```

and remove the conf for now

Reply:
Lets try and see if some test break...